CaptureImage.py

```diff
@@ -0,0 +1,47 @@
+import os
+import cv2
+
+def main() :
+
+	gestureClass = input("Enter the type of gesture")
+
+	cap = cv2.VideoCapture(0)
+
+	rootName = 'HandGestures/' + gestureClass
+
+	try:
+		if(not os.path.exists(rootName)):
+			os.makedirs(rootName)
+	except OSError:
+		print('Cannot create the directory')
```

Comment:
Are you sure that you shall allow all the code to run, although an exception has been thrown here. You might want to exit the script here (using something like `sys.exit(1)`.

Also, change the error message to be more clear: "Directory already exists."